src/server.h

```diff
@@ -347,8 +347,9 @@ extern int configOOMScoreAdjValuesDefaults[CONFIG_OOM_COUNT];
 
 /* RDB return values for rdbLoad. */
 #define RDB_OK 0
-#define RDB_NOT_EXIST 1 /* RDB file doesn't exist. */
-#define RDB_FAILED 2    /* Failed to load the RDB file. */
+#define RDB_NOT_EXIST 1    /* RDB file doesn't exist. */
+#define RDB_FAILED 2       /* Failed to load the RDB file. */
+#define RDB_INCOMPATIBLE 3 /* RDB version or signature is not compatible */
```

Comment:
Nit pick:

The macro ordering annoys me a bit. For me RDB_INCOMPATIBLE should come before RDB_FAILED to signal `RDB_FAILED` is a broader set of failure whereas `RDB_INCOMPATIBLE` covers only a small set of validation.


```suggestion
#define RDB_NOT_EXIST 1    /* RDB file doesn't exist. */
#define RDB_INCOMPATIBLE 2 /* RDB version or signature is not compatible */
#define RDB_FAILED 3       /* Failed to load the RDB file. */
```